modules/persistent-storage-csi-secrets-store-driver-uninstall.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-secrets-store-driver-uninstall_{context}"]
+= Uninstalling the Secrets Store CSI Driver Operator
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+* Administrator access to the cluster.
+
+.Procedure
+
+To uninstall the Secrets Store CSI Driver Operator:
+
+. Stop all application pods that use the `secrets-store.csi.k8s.io` provider.
+. Remove the Container Storage Interface (CSI) driver and associated manifests:
```

Comment:
We need to also point out that the provider also needs to be uninstall.
@dobsonj @lpettyjo

Reply:
Yes, IMO something like `Remove any third-party provider plugin for your chosen secret store` immediately after the `Stop all application pods` bullet.

Reply:
Done.